renderer/HardwareBaseGPU.cpp

```diff
@@ -0,0 +1,436 @@
+/*
+* Descent 3
+* Copyright (C) 2024 Parallax Software
```

Comment:
As this is new genuine code, there need add our own copyright header:

```
* Copyright (C) 2024 Descent Developers
```

Reply:
I will change it, although it's not really new, but a mix...